transformer_engine/pytorch/attention.py

```diff
@@ -5090,19 +5089,15 @@ def forward(
                 output.reshape(batch_size * max_seqlen_q // cp_size, -1).transpose_2d()
                 output = output.reshape(batch_size, max_seqlen_q // cp_size, -1)
```

Comment:
I think the logic for FP8 MHA is probably wrong here. Could you change it to something like this:
```
output = Float8Tensor.make_like(output, data=output._data.transpose(0,1).contiguous())
```

Reply:
It's ok, I'll address this in a different PR.